app/views/helpers/PlannedDowntimeTableMobile.scala.html

```diff
@@ -0,0 +1,55 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.Channel._
+@import models.PlannedDowntimes
+@import utils.DateTimeFormatter._
+
+@(plannedDowntimes: PlannedDowntimes)(implicit request: Request[_], messages: Messages)
+
+<table class="govuk-table pd-table-mobile">
+    <tbody class="govuk-table__body">
+    @for(downtime <- plannedDowntimes.plannedDowntimes) {
+
+    <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">@messages("planned-downtime.system.core.name")</th>
+        @if(downtime.affectedChannel.equals(gbArrivals)){
+        <td class="govuk-table__cell govuk-!-text-align-right">@messages("planned-downtime.ncts.gb.arrivals")</td>
+        }else if(downtime.affectedChannel.equals(xiArrivals)){
+        <td class="govuk-table__cell govuk-!-text-align-right">@messages("planned-downtime.ncts.xi.arrivals")</td>
+        }else if(downtime.affectedChannel.equals(gbDepartures)){
+        <td class="govuk-table__cell govuk-!-text-align-right">@messages("planned-downtime.ncts.gb.departures")</td>
+        }else if(downtime.affectedChannel.equals(xiDepartures)){
+        <td class="govuk-table__cell govuk-!-text-align-right">@messages("planned-downtime.ncts.xi.departures")</td>
+        }
+    </tr>
+    <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">@messages("planned-downtime.start")</th>
+        <td class="govuk-table__cell govuk-!-text-align-right">
+            @formatDate(downtime.startDate)<br />
+            @formatTime(downtime.startTime)
+        </td>
+    </tr>
+    <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">@messages("planned-downtime.end")</th>
+        <td class="govuk-table__cell govuk-!-text-align-right">
+            @formatDate(downtime.endDate)<br />
```

Comment:
not sure if we should use <br/> on the views

Reply:
`<br>`

Reply:
didn't know github actually lets you html in the comments!